pokemongo_bot/cell_workers/incubate_eggs.py

```diff
@@ -176,6 +180,9 @@ def _check_inventory(self, lookup_ids=[]):
                             pokemon.get('individual_stamina', 0)
                         ]})
                     matched_pokemon.append(pokemon)
+                    #remove as egg and add as pokemon
+                    inventory.pokemons().remove(pokemon['id'])
+                    inventory.pokemons().add(inventory.Pokemon(pokemon))
```

Comment:
It all depends on what the phone app is doing. If it only queries the inventory once, so should we. Otherwise it would be a big red flag for Niantic.